ribbon/src/main/java/feign/ribbon/LBClient.java

```diff
@@ -99,8 +99,12 @@ Request toRequest() {
           .request();
     }
 
+    Client client() {
```

Comment:
It was moved here as otherwise the LBClientFactory would have the take the delegate and the clientname and I wanted to be able to key the LBClient just off the clientName.